app/src/main/java/edu/uw/myuw_android/CommonWebViewFragment.kt

```diff
@@ -1,40 +1,49 @@
-package com.example.myuw
+package edu.uw.myuw_android
 
 import android.annotation.SuppressLint
 import android.content.Intent
-import android.graphics.Bitmap
 import android.net.Uri
 import android.os.Bundle
 import android.util.Log
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.webkit.WebResourceRequest
+import android.webkit.WebResourceResponse
 import android.webkit.WebView
 import android.webkit.WebViewClient
 import android.widget.LinearLayout
 import androidx.appcompat.app.AppCompatActivity
 import androidx.fragment.app.Fragment
-import androidx.navigation.Navigator
 import androidx.navigation.fragment.findNavController
 import androidx.navigation.fragment.navArgs
 import androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+import edu.my.myuw_android.BuildConfig
+import edu.my.myuw_android.R
+import net.openid.appauth.AuthorizationService
+import java.lang.Exception
+import java.net.URL
+import java.net.URLDecoder
+import java.nio.charset.StandardCharsets
 import java.util.*
+import kotlin.collections.HashMap
 
 private var webViewMap: MutableMap<String, WebView> = mutableMapOf()
 
 class CommonWebViewFragment: Fragment() {
-    val args: CommonWebViewFragmentArgs by navArgs()
+    private val args: CommonWebViewFragmentArgs by navArgs()
     lateinit var webView: WebView
+    lateinit var baseUrl: String
     lateinit var swipeRefreshLayout: SwipeRefreshLayout
+    private lateinit var authorizationService: AuthorizationService
 
     inner class CustomWebViewClient: WebViewClient() {
 
         override fun onPageFinished(view: WebView?, url: String?) {
             super.onPageFinished(view, url)
             swipeRefreshLayout.isRefreshing = false
 
-            (activity as AppCompatActivity).supportActionBar!!.title = webView.title.split(": ")[1]
+            (activity as? AppCompatActivity)?.supportActionBar!!.title = webView.title.split(": ").getOrElse(1){"Invalid Title"}
```

Comment:
Just wondering how can we make sure that supportActionBar is not null? 
Maybe we can break this line into two lines where one gets the supportActionBar and make sure it's not null and one gets the title?